proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -727,6 +820,83 @@ message BlindedBeaconBlockBodyDeneb {
+}
+
+message SignedBlindedBeaconBlockEpbs {
+    // The unsigned blinded beacon block itself.
+    BlindedBeaconBlockEpbs message = 1;
+
+    // 96 byte BLS signature from the validator that produced this blinded block.
+    bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message BlindedBeaconBlockEpbs {
+    // Beacon chain slot that this blinded block represents.
+    uint64 slot = 1 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.Slot"];
+
+    // Validator index of the validator that proposed the block header.
+    uint64 proposer_index = 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.ValidatorIndex"];
+
+    // 32 byte root of the parent block.
+    bytes parent_root = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+
+    // 32 byte root of the resulting state after processing this blinded block.
+    bytes state_root = 4 [(ethereum.eth.ext.ssz_size) = "32"];
+
+    // The blinded beacon block body.
+    BlindedBeaconBlockBodyEpbs body = 5;
+}
+
+message BlindedBeaconBlockBodyEpbs {
+    // The validators RANDAO reveal 96 byte value.
+    bytes randao_reveal = 1 [(ethereum.eth.ext.ssz_size) = "96"];
+
+    // A reference to the Ethereum 1.x chain.
+    Eth1Data eth1_data = 2;
+
+    // 32 byte field of arbitrary data. This field may contain any data and
+    // is not used for anything other than a fun message.
+    bytes graffiti = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+
+    // At most MAX_PROPOSER_SLASHINGS.
+    repeated ProposerSlashing proposer_slashings = 4 [(ethereum.eth.ext.ssz_max) = "16"];
+
+    // At most MAX_ATTESTER_SLASHINGS.
+    repeated AttesterSlashing attester_slashings = 5 [(ethereum.eth.ext.ssz_max) = "2"];
+
+    // At most MAX_ATTESTATIONS.
+    repeated Attestation attestations = 6 [(ethereum.eth.ext.ssz_max) = "128"];
+
+    // At most MAX_DEPOSITS.
+    repeated Deposit deposits = 7 [(ethereum.eth.ext.ssz_max) = "16"];
+
+    // At most MAX_VOLUNTARY_EXITS.
+    repeated SignedVoluntaryExit voluntary_exits = 8 [(ethereum.eth.ext.ssz_max) = "16"];
+
+    // Sync aggregate object for the beacon chain to track sync committee votes. New in Altair network upgrade.
+    SyncAggregate sync_aggregate = 9;
+
+    // At most MAX_BLS_TO_EXECUTION_CHANGES. New in Capella network upgrade.
+    repeated SignedBLSToExecutionChange bls_to_execution_changes = 10 [(ethereum.eth.ext.ssz_max) = "16"];
+
+    repeated SignedExecutionPayloadHeaderEnvelope signed_execution_payload_header_envelope = 11;
+
+    // At most MAX_PAYLOAD_ATTESTATIONS. New in Epbs network upgrade.
+    repeated PayloadAttestation payload_attestations = 12 [(ethereum.eth.ext.ssz_max) = "2"];
+
+    // At most MAX_EXECUTION_LAYER_WITHDRAW_REQUESTS. New in Epbs network upgrade.
+    repeated ExecutionLayerWithdrawalRequest execution_payload_withdraw_requests = 13 [(ethereum.eth.ext.ssz_max) = "16"];
+}
```

Comment:
We shouldn't need to worry about blind bock anymore. That's the essence of epbs

Reply:
Note for DB storage, we'll store block + header by default. The payload will be on the side but by default that's requested through EL client anyway